docs/book/en/src/start/docker.md

```diff
@@ -100,9 +100,9 @@ The `wasmedge/wasmedge:release-{version}` Docker images provide a slim WasmEdge
 Use `wasmedgec` and `wasmedge` ([link](https://github.com/WasmEdge/WasmEdge/tree/master/examples/wasm)):
 
 ```bash
-$ docker pull wasmedge/wasmedge:release-0.10.1
```

Comment:
The version should update to `0.11.0`

Reply:
Ok, I'll use `mdbook-variables` preprocessor to make all versions in documents configurable.